proposals/0000-swiftpm-script-support.md

```diff
@@ -123,12 +125,26 @@ SwiftPM will utilize the changes to the compiler front-end to access packages re
 
 Once package dependencies are known, SwiftPM will use it's existing logic for resolving those dependencies and outputting a ".resolved" file to ensure future invocations are reproducible.
 
-The name of the resolved file will be `script_name.resolved`.
+The name of the resolved file will be `script_name.resolved`. By default, the resolve file will be located under `~/.swiftpm/scripts` but can be specified to another path if desired.
```

Comment:
Many people on the forums brought up the point about this proposal not using standard directories for storing user cache data. We should just update the proposal to use the standard ones.